xgi/drawing/draw.py

```diff
@@ -3,11 +3,12 @@
 from inspect import signature
 from itertools import combinations
 
+import matplotlib as mpl
 import matplotlib.pyplot as plt
 import numpy as np
 import seaborn as sb  # for cmap "crest"
 from matplotlib import cm
-from matplotlib.patches import FancyArrow
+from matplotlib.patches import FancyArrow, FancyArrowPatch
```

Comment:
Looks like we don't need anymore the `FancyArrow` import, right?

Reply:
Good catch, done